files/build_templates/sonic_debian_extension.j2

```diff
@@ -319,6 +319,16 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i {{deb}} || sudo LANG=C DEBIAN_FRONTEND=noni
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-db-cli_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install sonic-rsyslog-plugin
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-rsyslog-plugin_*.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
+
+# Generate host conf for rsyslog_plugin
+j2 -f json $BUILD_TEMPLATES/rsyslog_plugin.conf.j2 $BUILD_TEMPLATES/events_info.json > $BUILD_TEMPLATES/host_events.conf
+sudo mv $BUILD_TEMPLATES/host_events.conf $FILESYSTEM_ROOT_ETC/rsyslog.d/ 
```

Comment:
>mv [](http://example.com/codeflow?start=5&length=2)

mv -> cp ?

Reply:
Instead of cp and deleting from build_templates, it made more sense just to move

Reply:
I see. Suggest let above command output to final target directory directly.

Reply:
Tried to do so, but even with sudo, I get permission denied error when outputting to destination directory.

Reply:
Got it. `sudo j2` does not work, you need pipe `j2 | sudo tee FILE`

Reply:
Done